### DISCUSSION TITLE
api call bug:

### DISCUSSION THREAD

**Discussion by ouyoung33** [jaredhendrickson13/pfsense-api#531] [General]
api call bug: When I manually create dnat,Nat will default "filter rule association" to "rule nat" to implement forwarding and connect to my internal machine; But when I create a dnat rule using the api, the "filter rule association" is "pass" but no forwarding is actually implemented, and I can't connect to my internal machine through dnat
![Uploading 微信截图_20240805112258.png…]()

**Comment by jaredhendrickson13** [comment]
Duplicate of issue #533
